media/js/mozorg/contact-spaces.js

```diff
@@ -115,12 +115,17 @@
             //set the current tab navigation item
             $('ul.category-tabs li[data-id="' + tab + '"]').addClass('current');
 
-            // set the current list menu navigation item
-            $('.nav-category li[data-id="' + id + '"]').addClass('current');
+            if (tab === 'spaces') {
+                // set the current list menu navigation item for the active space
+                $('.nav-category li[data-id="' + id + '"]').addClass('current');
+            }
 
-            // hide the community sub menu's
-            $('.accordion .submenu').hide();
-            mozMap.toggleCommunitySubMenu();
+            if (tab === 'communities') {
+                // hide the community sub menu's
+                $('#nav-communities').hide();
```

Comment:
Ah, but this still has to target the child `.submenu`'s. Should have been more clear in my previous comment.